src/components/Fields/TextField.test.js

```diff
@@ -0,0 +1,308 @@
+/* global describe, expect, it, shallow, mount */
+import React from 'react';
+import sinon from 'sinon';
+import TextField from './TextField';
+import Tooltip from '../Tooltip/Tooltip';
+import TooltipBox from '../Tooltip/TooltipBox';
+import { TooltipBoxStatus } from '../Tooltip/TooltipEnums';
+import IconClear from '../Icon/IconClear';
+import IconSearch from '../Icon/IconSearch';
+import IconRequired from '../Icon/IconRequired';
+import Button from '../Button/Button';
+
+function createTextfield() {
+    return shallow(<TextField label="My Input" />);
+}
+
+describe('TextField', () => {
+    it('renders an div wrapper element', () => {
```

Comment:
Could you please use the new isConformant helper in this file to ensure correct classes, etc.?